<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPrintPackList</class>
 <widget class="QDialog" name="DialogPrintPackList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Упаковочный лист</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>610</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>430</width>
       <height>610</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPrint">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
